### PR TITLE
### Fixed- `contracts.createUnsignedRunMessage` must migrate public key to header.pubkey when possible.

### PR DESCRIPTION
**Pull request by melsomino**

### CODE REVIEW

File: __tests__/contracts.js [hunk 1]
```diff
@@ -1044,3 +1044,96 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
     expect(checked.alreadyDeployed)
         .toBeTruthy();
 });
+
+
+const SensorPackage = {
```
Comment (AlexeyVavilin) [inline __tests__/contracts.js:1049]: No packages in code. Move to file as all other contracts

File: __tests__/contracts.js [hunk 2]
```diff
@@ -1044,3 +1044,96 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
...
+        constructorParams: { ownerKey: `0x${ownerKeys.public}` },
+        keyPair: deployKeys,
+    });
+
+    const futureAddress = deployMessage.address;
+
+    // const futureAddress = '0:03285608adf64c1db67cb7f970271d1a6d83a75fa5f73e8778a08afd1be4fd8d'
+    console.log(`Future address of the contract will be: ${futureAddress}`);
+
+    // Requesting contract deployment funds form a local TON OS SE giver
```
Comment (AlexeyVavilin) [inline __tests__/contracts.js:1113]: I don't understand this comment. What does it mean?

File: __tests__/contracts.js [hunk 3]
```diff
@@ -1044,3 +1044,96 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
...
+        keyPair: deployKeys,
+    })).address;
+
+
+    await client.contracts.run({
+        address: helloAddress,
+        abi: SensorPackage.abi,
+        functionName: 'setData',
+        header: {
+            pubkey: ownerKeys.public,
```
Comment (AlexeyVavilin) [inline __tests__/contracts.js:1131]: Isn't this test for automatic adding pubkey to header? Maybe remove this line?

**Review by AlexeyVavilin — CHANGES_REQUESTED** [review]

File: __tests__/contracts.js (hunk 3 above)
Comment (melsomino) [reply __tests__/contracts.js:1131]: It is a part of test. People want to deploy with owner key different from deploy keys.

File: .env [hunk 4]
```diff
@@ -1,4 +1,4 @@
 USE_NODE_SE=true
-TON_NETWORK_ADDRESS=http://0.0.0.0:8080
+TON_NETWORK_ADDRESS=http://0.0.0.0:8083
```
Comment (g9d) [inline .env:2]: Looks like this change is not related to commit

File: __tests__/contracts.js (hunk 1 above)
Comment (melsomino) [reply __tests__/contracts.js]: done

File: __tests__/contracts.js (hunk 2 above)
Comment (melsomino) [reply __tests__/contracts.js]: This code is inherited from support team. So I just have removed this comment.

File: .env (hunk 4 above)
Comment (melsomino) [reply .env:2]: this file is used only by developers

**Review by AlexeyVavilin — APPROVED** [review]
